### ISSUE TITLE
Unable to sync/scrobble (401, 502, 504 errors)

### ISSUE THREAD

**Issue by fuzeman** [trakt/Plex-Trakt-Scrobbler#157]
**Unknown - "Unknown" (code: 502 or 504)**
This error likely indicates trakt.tv API server problems, no "fix" is available for the plugin. We will just need to wait until these problems have been resolved.

**Unauthorized - "OAuth must be provided" (code: 401)**
I'm unsure exactly what is causing this issue, it is either:
 - The plugin hasn't been able to authenticate properly (try restarting Plex Media Server, so the plugin tries to re-authenticate)
 - Otherwise, this could be another issue with the API?

Please send your full ```com.plexapp.plugins.trakttv.log``` file if you experience this issue, as I haven't been able to reproduce this issue myself.

**Comment by scottrobertson** [comment]
Hey @fuzeman, just following up from the issue in #153. I have restarted, here is a full log:

https://gist.github.com/scottrobertson/160ae99caaa5a6b72460

**Comment by tomasswood** [comment]
I have this issue as well. I couldn't find my logs, but here is my request body:

https://gist.github.com/e8eb82847f1c90e98250.git

**Comment by StErMi** [comment]
I think that Trakt has made some huge changes in the latest days with Trakt 2.0 and now they are full of bugs and problems.

For istance, I cannot create a custom list on the site anymore...

**Comment by scottrobertson** [comment]
Lists are coming back soon, very few people used them so they didn't want to slow v2 down by waiting on them.

**Comment by StErMi** [comment]
@scottrobertson I'm using them for FlexGet ;)

**Comment by scottrobertson** [comment]
Just use Sonarr, no need for any of that :p

**Comment by StErMi** [comment]
Mmm cool, what's the main differences?

**Comment by fuzeman** [comment]
I've found the plugin wasn't re-authenticating correctly if you changed your username, this caused a 401 error on any requests.

**v0.9.1.6** (https://forums.plex.tv/index.php/topic/102818-rel-trakt/page-27#entry818966) has a fix for this if anyone still has this error code coming up.

**Comment by shadone** [comment]
Confirming that v0.9.1.6 solves the issue for me, thanks!

**Comment by mjryan1** [comment]
I'm not sure if what I'm getting is the same error. I've been searching through the post and trying different things that have been posted with no luck.
 basically I get invalid authentication. after sever different attempts to fix uninstalling and reinstalling the latest version beta. at on point it would tell me tried to sync 2min ago and failed 2sec something like that.

Here is my log file:
https://www.dropbox.com/sh/e7uvh8yz6uphh17/AADvjOsa_L9ze2fqEJVJjIcea?dl=0

Thanks for any help.

**Comment by fuzeman** [comment]
@mjryan1 the trakt.tv API seems to be down currently, hopefully it will come back online soon.

**Comment by JBerkvens** [comment]
I don't know if it helps, but I also get the 502 error but it doesn't say: Unknown - "Unknown".
I get the error message: failed (trakt.tv - 502: "server unavailable").

**Comment by Wolf666** [comment]
with 0.9.1.10 got 502 with "server unavailable"
I am also running Plex Preview 0.9.11.9
